backend/Tasque.Core.WebAPI/Controllers/TaskController.cs

```diff
@@ -75,5 +75,12 @@ public async Task<IActionResult> GetTasksType()
             return Ok();
             //   return Ok(await _taskService.GetTasksType());
```

Comment:
do we need this commented part?

Reply:
We have not implemented the required method yet. I've added TODO to this comment.